app/_data/docs_nav_kuma_dev.yml

```diff
@@ -117,6 +119,15 @@ items:
             url: /production/secure-deployment/certificates/
       - text: Kuma user interface
```

Comment:
```suggestion
      - text: {{site.mesh_product_name}} user interface
```

Reply:
I'm not sure we are able to use templates in this file